Sources/StructuredQueriesCore/ColumnGroup.swift

```diff
@@ -0,0 +1,92 @@
+/// A group of table columns.
+///
+/// Don't create instances of this value directly. Instead, use the `@Table` and `@Columns` macros
+/// to generate values of this type.
+@dynamicMemberLookup
+public struct ColumnGroup<Root: Table, Values: Table>: _TableColumnExpression
```

Comment:
Bikeshed opportunity: Should we call this `TableColumnGroup` for consistent prefixing?